src/types/venues.ts

```diff
@@ -17,6 +18,9 @@ import { UpcomingEvent } from "./UpcomingEvent";
 import { User, UserStatus } from "./User";
 import { VenueAccessMode } from "./VenueAcccess";
 import { VideoAspectRatio } from "./VideoAspectRatio";
+import { WorldSlug } from "./world";
+
+export type SpaceSlug = Brand<string, "SpaceSlug">;
```

Comment:
Maybe an analogy to `WithId`:

```suggestion
export type SpaceSlug = WithBrand<string, "SpaceSlug">;
```